docs/subsystems/icons.md

```diff
@@ -30,28 +30,137 @@ using icons. Typically, this means:
+   visual editor for vector images, including SVG.
+
+1. Use Ctrl + A or otherwise select all components of the icon.
+
+1. Locate Inkscape's Path menu, and choose the Stroke to Path item. You'll
+   immediately see your icon appear to balloon up: this is expected, because
+   the previous stroke in use has now been applied to your icon's converted paths.
+
+1. Examine the Stroke style applied to your paths on the Fill and Stroke tab of
+   Inkscape's object panel. Be sure to reduce the Stroke to 0:
+
+| Stroke incorrectly still applied                                             | Stroke set to zero                                                                       |
+| ---------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| ![Ballooned-out icon shapes](../images/icon-prep-stroke-to-path-initial.png) | ![Expected icon shapes without stroke](../images/icon-prep-stroke-to-path-completed.png) |
+
+At this point, you can save your file in the `web/shared/icons/` directory and
+close Inkscape.
+
+While Inkscape does a fine job with the stroke-to-path conversion, it leaves a lot
+of crufty markup behind. So next, open your icon's SVG file in your editor of choice.
+
+Once opened in your editor, the file's Inkscape source will something like this (this
+example shows only the first 25 lines or so):
+
+```xml
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="24"
+   height="24"
+   viewBox="0 0 24 24"
+   fill="none"
+   stroke="currentColor"
+   stroke-width="2"
+   stroke-linecap="round"
+   stroke-linejoin="round"
+   class="lucide lucide-expand-icon lucide-expand"
+   version="1.1"
+   id="svg8"
+   sodipodi:docname="expand.svg"
+   inkscape:version="1.4 (e7c3feb1, 2024-10-09)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs8" />
+  <sodipodi:namedview
+     id="namedview8"
+     pagecolor="#ffffff" />
+
+<!-- ...and so on. -->
+```
+
+Your last task is to clean up the SVG's XML so that only the essential components
+remain:
+
+1. Consider using Git to add and temporarily commit the icon as output by Inkscape.
+   If you make a mistake while cleaning up the icon, you can use `git restore` to
+   return to that point.
+
+1. Strip back the opening of the file to include only the `<svg>` tag and its `xmlns`,
+   `width`, `height`, `viewBox`, and `fill` attributes and their values as set by
+   Inkscape. The one exception there is the `fill` attribute's value, which you should
+   set to `#000000`: while that has no bearing on how the icon will render in Zulip, it
+   makes it easy to see the icon in operating-system previews and elsewhere.
```

Comment:
`fill` is unnecessary. The [initial value](https://www.w3.org/TR/SVG2/painting.html#SpecifyingFillPaint) is already black.